### PR TITLE
Moved ROS1 from proxy submodule into ros1 submodule

### PR DESCRIPTION
**Pull request by schmerl**

### CODE REVIEW

File: src/roswire/proxy/__init__.py [hunk 1]
```diff
@@ -1,2 +1 @@
 # -*- coding: utf-8 -*-
```
Comment (ChrisTimperley) [inline src/roswire/proxy/__init__.py:1]: We can remove the `proxy` directory entirely.

File: src/roswire/ros1/__init__.py [hunk 2]
```diff
@@ -1,7 +1,2 @@
 # -*- coding: utf-8 -*-
-from .node import ROS1Node
```
Comment (ChrisTimperley) [inline src/roswire/ros1/__init__.py:2]: These lines shouldn't be removed.

File: src/roswire/ros1/ros1.py [hunk 3]
```diff
@@ -74,7 +77,7 @@ def __init__(self,
                            self.__shell)
         self.__state_probe: SystemStateProbe = \
             SystemStateProbe.via_xmlrpc_connection(self.__connection)
-        self.roslaunch: ROSLaunchManager = \
+        self.roslaunch: 'ROSLaunchManager' = \
```
Comment (ChrisTimperley) [inline src/roswire/ros1/ros1.py:80]: No need for quotes.

File: test/test_basic.py [hunk 4]
```diff
@@ -10,7 +10,7 @@
 import roswire.exceptions
 from dockerblade import Shell, FileSystem
 from roswire import AppInstance, ROSWire, AppDescription
-from roswire.proxy import ROS1
+from roswire.ros1 import ROS1
```
Comment (ChrisTimperley) [inline test/test_basic.py:13]: We could add a `ROS1` and `ROS2` import to `roswire/__init__.py`?

**Comment by schmerl** [comment]
Made all the changes suggested

File: src/roswire/ros1/__init__.py (hunk 2 above)
Comment (ChrisTimperley) [reply src/roswire/ros1/__init__.py:2]: Looks like this line is still missing?

**Review by ChrisTimperley — APPROVED** [review]

**Pull request merged**